nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/model/PropertyException.java

```diff
@@ -37,16 +37,44 @@ public PropertyException() {
         super();
     }
 
+    /**
+     * @since 11.1
+     */
+    public PropertyException(int statusCode) {
+        super(statusCode);
+    }
+
     public PropertyException(String message) {
         super(message);
     }
 
+    /**
+     * @since 11.1
+     */
+    public PropertyException(String message, int statusCode) {
+        super(message, statusCode);
+    }
+
     public PropertyException(String message, Throwable cause) {
         super(message, cause);
     }
 
+    /**
+     * @since 11.1
+     */
+    public PropertyException(String message, Throwable cause, int statusCode) {
+        super(message, cause, statusCode);
+    }
+
     public PropertyException(Throwable cause) {
         super(cause);
     }
 
+    /**
+     * @since 11.
+     */
+    public PropertyException(Throwable cause, int statusCode) {
+        super(cause, statusCode);
+    }
+
```

Comment:
Do you really need to add these constructors on this deprecated exception? Are they used somewhere?

Reply:
I do need to add them because I want to give a `statusCode` to `ReadOnlyPropertyException` which extends `PropertyException` and deprecated `PropertyException`.